api/actions.json

```diff
@@ -99,5 +99,27 @@
     "fromBranchTransactionId": "PgR3qYexQoOSHzprXbBT"
 }
 
+-----------------------------------------------------------------------------------------------------------------------------------
+-----------------------------------------------------------------------------------------------------------------------------------
+
+# Move product from headoffice to branch (POST)
+# Endpoint: http://localhost:5001/local-parikrama/us-central1/api/api/actions/moveProduct
+#Request
+{
+	"fromBranch": "MxoS2K8t8jT7MATniD4x",
+	"toBranch": "R1vTnxSLWByLa7NocDDX",
+	"fromBranchName": "sarjapur",
+	"toBranchName": "madiwala",
+	"productId": "3V0TKHqOC2GLzzNceFKs",
+	"productName": "blankets",
+	"operationalQuantity": 10
+}
+
+#Response
+{
+    "toBranchTransactionId": "v8jszk47HSPQP2IG1a7w",
+    "fromBranchTransactionId": "PgR3qYexQoOSHzprXbBT"
+}
```

Comment:
will it make sense to add product Id as well in the response ?

Reply:
no i guess, actually even these trxIds are not needed. Added them just in case UI needs.